a2c_ppo_acktr/envs.py

```diff
@@ -95,16 +95,16 @@ def make_vec_envs(env_name,
         make_env(env_name, seed, i, log_dir, allow_early_resets, **kwargs)
         for i in range(num_processes)
     ]
-    if len(envs) > 1:
-        envs = SubprocVecEnv(envs)
-    else:
-        envs = DummyVecEnv(envs)
-
-    if len(envs.observation_space.shape) == 1:
-        if gamma is None:
-            envs = VecNormalize(envs, norm_reward=False)
-        else:
-            envs = VecNormalize(envs, gamma=gamma)
+    # if len(envs) > 1:
```

Comment:
Why did you remove the normalization?

Reply:
You're right - the normalization should be here. I thought at the beginning that for this experiment we don't need it, but for the recurrent experiment the normalization improves performance. It needs to be there.